### ISSUE TITLE
Icon for "Folder for periodic saving" button doesn't show up properly

### ISSUE THREAD

**Issue by Vovkiv** [vikdevelop/SaveDesktop#95]
![image](https://github.com/vikdevelop/SaveDesktop/assets/54743395/340543b9-ef13-49ef-a36c-8df2910639b5)

Icon for "Folder for periodic saving" button doesn't show up, so placeholder instead. Flatpak version, 2.6.1. I checked commits for 2.6.2 and didn't saw there anything related to icon, so i think same will happens with 2.6.2.

Also, on import page, it shows folder icon instead of arrow:
![image](https://github.com/vikdevelop/SaveDesktop/assets/54743395/28e08dea-65d6-4d29-be36-2b7bcbbc6ef3)
While screenshots on github shows this:
![image](https://raw.githubusercontent.com/vikdevelop/SaveDesktop/main/flatpak/screenshots/import_page_dark.png#gh-dark-mode-only)
![image](https://raw.githubusercontent.com/vikdevelop/SaveDesktop/main/flatpak/screenshots/main_window_dark.png#gh-dark-mode-only)

**Comment by vikdevelop** [comment]
Hello,
As for the icon to select the folder for periodic saving, I'll add that to the `flatpak/symbolic-icons` directory.
As for the other icon, it's different because you probably have a different icon theme installed.
Since I was unable to release a new version 2.6.2 on Flathub, these changes will be tied to that version.

**Comment by Vovkiv** [comment]
> As for the other icon, it's different because you probably have a different icon theme installed.

The thing is: i don't have anything custom installed. No custom themes, iconpacks, even gnome extensions.

**Comment by vikdevelop** [comment]
I've released a new version on Flathub with a fixed icon for selecting the periodic saving folder. As for the other icon, it is tied to the icon theme (I use the Tela circle icon).

**Comment by vikdevelop** [comment]
In my opinion, there is no reason to create new screenshots, since the current version is not that different from the version shown in the screenshot (2.5), the only difference is a different icon next to the button for selecting the interval of periodic saving. The icon on the configuration import page that is in your screenshot is not much different from the icon that is in the screenshot here on GitHub, they basically indicate the same thing, in my opinion it's a trivial problem. 
I'll definitely release new screenshots in the future, but certainly not at the moment, when the differences between versions are minimal, as I wrote above.

**Comment by Vovkiv** [comment]
> In my opinion, there is no reason to create new screenshots, since the current version is not that different from the version shown in the screenshot (2.5), the only difference is a different icon next to the button for selecting the interval of periodic saving. The icon on the configuration import page that is in your screenshot is not much different from the icon that is in the screenshot here on GitHub, they basically indicate the same thing, in my opinion it's a trivial problem. I'll definitely release new screenshots in the future, but certainly not at the moment, when the differences between versions are minimal, as I wrote above.

Yes, it's not big deal, but it's just strange, when program page advertise UI, that will be different from UI then what end user will get after they install said program, even if it's different icons somewhere.
Here at https://stopthemingmy.app/ you could read about why people behind "Please don’t theme our apps" thinks it's bad practice.
